test/type-resolution/__snapshots__/fixtures.test.ts.snap

```diff
@@ -941,10 +927,10 @@ TypedTemplate {
   "resources": DependencyGraph {
     "_dependencies": Map {
       "VPC" => Set {},
+      "MyTaskDef" => Set {},
       "Cluster" => Set {
         "VPC",
       },
-      "MyTaskDef" => Set {},
```

Comment:
Order of `_dependencies` is an implementation detail of set. They are now in the topological order due to a change in the way that particular graph is created.